SpaceHorse/SpaceHorse/main.cpp

```diff
@@ -39,12 +39,10 @@ int main(int argc, char* argv[])
 	//setup display
 	ALLEGRO_DISPLAY_MODE disp_data;
 	al_get_display_mode(al_get_num_display_modes() - 1, &disp_data); //Set resolution to max
-	//windowWidth = backgroundWidth;
-	//windowHeight = backgroundHeight;
-	//windowWidth = 700;
-	//windowHeight = 700;
-	int windowWidth = disp_data.width;
-	int windowHeight = disp_data.height;
+	int windowWidth = 1600;

```

Comment:
Why does this have to change here? It works better when you use disp_data.width etc to set the window dimensions. I just have those other values for debugging. I know there is a problem with hardcoded background resolution though, will have to prep a story to bring scaling into the display logic somewhere, so that all native resolutions look right.